src/matrixeventprocessor.ts

```diff
@@ -85,93 +92,29 @@ export class MatrixEventProcessor {
     }
 
     public async EventToEmbed(
-        event: IMatrixEvent, profile: IMatrixEvent|null, channel: Discord.TextChannel,
+        event: IMatrixEvent, channel: Discord.TextChannel, getReply: boolean = true,
     ): Promise<IMatrixEventProcessorResult> {
-        let body: string = this.config.bridge.disableDiscordMentions ? event.content!.body as string :
```

Comment:
Are we no longer mention matching?

Reply:
Nope, we had discussed this in the matrix room, to only use pills for user mentions.
![screenshot_2018-11-30_11-02-29](https://user-images.githubusercontent.com/2433620/49282445-7e398680-f48f-11e8-8e75-05a6ea984c44.png)